app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    func_cache = {}
+
+    def wrapper(*args) -> Callable:
+        if args in func_cache:
```

Comment:
you should check not only `args` but also `kwargs` together
combine `args`  and `kwargs`  into one key